repl.html

```diff
@@ -31,18 +31,19 @@
 </div>
 <div>
 <!-- TODO(max): Add button to save to/load from disk. -->
-<button id="clear-local-storage">Clear LocalStorage</button>
+<button id="clear-local-storage">Clear</button>
 </div>
 <div id="output" style="height: 400px; overflow: auto;">
 Output:
 </div>
 <div>
-<code>>>> </code><input id="input" type="text" />
+<code><span id="prompt-string">>>></span></code><input id="input" type="text" />
```

Comment:
Not sure I understand. Aren't there more than 1 `>>>` span? Then there should not be multiple elements of the same ID

Reply:
The way it's currently coded there's only one prompt string ">>>" next to the input element

Reply:
`updateHistory` adds more `>>>`

Reply:
Those are styled differently because they are part of `.language-text`, but for the main input prompt string we need a different CSS selector, otherwise it doesn't get the correct font style.

Reply:
oh, sorry, totally misread. thank you